src/sentry/api/endpoints/project_platforms.py

```diff
@@ -1,18 +1,34 @@
 from __future__ import absolute_import
 
+from rest_framework import serializers, status
 from rest_framework.response import Response
 from sentry.api.bases.project import ProjectEndpoint
-from sentry.api.serializers import serialize, register, Serializer
+from sentry.api.serializers import serialize
 from sentry.models import ProjectPlatform
 
 
-@register(ProjectPlatform)
-class ProjectPlatformSerializer(Serializer):
-    def serialize(self, obj, attrs, user):
-        return {'platform': obj.platform, 'dateCreated': obj.date_added}
+ERR_FIELD_REQUIRED = 'This field is required.'
+
+
+class ProjectPlatformSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ProjectPlatform
+        fields = ('platform',)
 
 
 class ProjectPlatformsEndpoint(ProjectEndpoint):
     def get(self, request, project):
         queryset = ProjectPlatform.objects.filter(project_id=project.id)
         return Response(serialize(list(queryset), request.user))
+
+    def put(self, request, project):
+        serializer = ProjectPlatformSerializer(data=request.DATA)
+
+        if not serializer.is_valid():
+            return Response(serializer.errors, status.HTTP_400_BAD_REQUEST)
+
+        platform = serializer.object
+        platform.project_id = project.id
+        platform.save()
```

Comment:
Will need to still handle the `IntegrityError` here that may be raised due to it being a duplicate.